packages/taquito/src/wallet/receipt.ts

```diff
@@ -2,9 +2,12 @@ import { OperationContentsAndResult } from '@taquito/rpc';
 import BigNumber from 'bignumber.js';
 import { flattenOperationResult } from '../operations/operation-errors';
 
+export const COST_PER_BYTE = 250;
```

Comment:
this is a protocol constant right? should we collect values like this in a constants file? so its easier to find and maintain.

although it wont be changing without a fair amount of forewarning

Reply:
Thanks, moved to the constants.ts file